src/api/get.test.js

```diff
@@ -3,21 +3,25 @@ import { get, getJSON } from './get'
 window.fetch = jest.fn()
 
 describe('get', () => {
+    it('rejects with the expected error message if baseUrl is not specified', () => {
+        return expect(get(undefined, 'path')).rejects.toMatchSnapshot()
+    })
+
     it('rejects with the expected error message if path is not specified', () => {
-        return expect(get()).rejects.toMatchSnapshot()
+        return expect(get('baseUrl', undefined)).rejects.toMatchSnapshot()
     })
 
     it('calls fetch with the expected path', () => {
         const actual = 'path'
```

Comment:
This use of actual is confusing to me. Usually you'd use actual for the actual result you're getting from the test (vs. the expected result).

Actual in the context of this test would be `get('baseUrl', 'path')` to my expectations.

Reply:
yeah, let's put this in a reminder issue.